src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -15617,7 +15617,15 @@ namespace N1
    class A { public int Foo() { return 2; }}
 }
 ";
-            CreateCompilationWithMscorlib(text).VerifyDiagnostics();
```

Comment:
I didn't specifically check, but I'm reasonably confident that we have at least one other test of namespace declarations in regular C#.